From 9a3372d695e0374821a0db6275b0b5c57111a341 Mon Sep 17 00:00:00 2001
From: Luke Wilde <lukew@serenityos.org>
Date: Wed, 13 Apr 2022 17:22:56 +0100
Subject: [PATCH 1/2] Remove use of futime(n)s

We do not currently support futimens or futimes. [futimens is a POSIX function,](https://pubs.opengroup.org/onlinepubs/9699919799/)
so this is an issue on our side.

- [ ] Local?
- [ ] Should be merged to upstream?
- [X] Resolves issue(s) with our side of things
- [ ] Hack
---
 src/util/vfs/vfs-fd.c | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/src/util/vfs/vfs-fd.c b/src/util/vfs/vfs-fd.c
index c15ab5c..8d8d5c8 100644
--- a/src/util/vfs/vfs-fd.c
+++ b/src/util/vfs/vfs-fd.c
@@ -200,11 +200,6 @@ static bool _vfdSync(struct VFile* vf, void* buffer, size_t size) {
 	UNUSED(size);
 	struct VFileFD* vfd = (struct VFileFD*) vf;
 #ifndef _WIN32
-#ifdef __HAIKU__
-	futimens(vfd->fd, NULL);
-#else
-	futimes(vfd->fd, NULL);
-#endif
 	if (buffer && size) {
 		return msync(buffer, size, MS_ASYNC) == 0;
 	}
-- 
2.36.1

